ads/README.md

```diff
@@ -44,9 +44,9 @@ More information can be provided in a similar fashion if needed (Please file an
 
 ### Methods available to the ad.
 
-- `window.context.noContentAvailable` is a function that the ad system can call if the ad slot was not filled. The container page will then react by showing fallback content or collapsing the ad if allowed by AMP resizing rules.
+- `window.context.renderStart(opt_data)` is a function that the ad system should call if the ad start rendering. The ad would then set the visibility of the iframe to visible. The ad type needs to be included in `waitForRenderStart` list in [integration.js](../3p/integration.js) for this function to be available. Ad system can use `opt_data` object of form `{width, height}` to send the returned ad size to request a resize. Please check [Ad resizing](https://github.com/ampproject/amphtml/blob/master/ads/README.md#ad-resizing) for more information.
+- `window.context.noContentAvailable` is a function that the ad system should call if the ad slot was not filled. The container page will then react by showing fallback content or collapsing the ad if allowed by AMP resizing rules.
```

Comment:
nit: since you added () to renderStart, please also make it for the other 2 methods.